recipes/grammarly/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "grammarly",
+  "name": "Grammarly",
+  "version": "0.1.0",
+  "license": "MIT",
+  "config": {
+    "serviceURL": "https://app.grammarly.com/docs/new"
```

Comment:
Please confirm that this url is correct. Since I do not have a grammarly account, I'm unable to verify this.